close-iteration/Main.hs

```diff
@@ -110,7 +108,7 @@ updateCompletedPoints projectId tasks =
                   <> display (storyUrl projectId story)
                   <> " to "
                   <> display completedPoints
-              _ -> logWarn "'points completed' field has incorrect type."
+              _ -> error "impossible"
```

Comment:
Kind of annoying, but `extractNumberField` can't return a non-`CustomNumber`. This is a quick and easy way to tag that.